distributed/src/main/scala/asobu/distributed/service/Extractors.scala

```diff
@@ -0,0 +1,200 @@
+package asobu.distributed.service
+
+import asobu.distributed.service.Action.DistributedRequest
+import asobu.distributed.service.Extractors._
+import asobu.distributed.RequestExtractorDefinition
+import asobu.distributed.gateway.SyncedExtractResult
+import asobu.distributed.service.Action.DistributedRequest
+import asobu.distributed.service.Extractors.{RouteParamsExtractor, BodyExtractor, RemoteExtractor}
+import asobu.dsl._
+import asobu.dsl.extractors.JsonBodyExtractor
+import asobu.dsl.util.HListOps.{RestOf, CombineTo, RestOf2}
+import cats.{Monad, Functor, Eval}
+import cats.sequence.RecordSequencer
+import shapeless.ops.hlist.Prepend
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Use brackets to group imports from the same package](https://www.codacy.com/app/iHeartRadio/asobu/file/2307107158/issues/source?bid=3037259&fileBranchId=3178877#l14)